diff -up kalgebra-4.10.1/CMakeLists.txt.opengl_optional kalgebra-4.10.1/CMakeLists.txt
--- kalgebra-4.10.1/CMakeLists.txt.opengl_optional	2013-03-01 00:40:03.799142005 -0600
+++ kalgebra-4.10.1/CMakeLists.txt	2013-03-18 18:00:31.348408921 -0500
@@ -11,8 +11,11 @@ find_library(QTMML_LIBRARY qtmmlwidget)
 include_directories(${CMAKE_SOURCE_DIR} ${CMAKE_BINARY_DIR} ${KDE4_INCLUDES} ${LIBKDEEDU_INCLUDE_DIR}/qtmml ${ANALITZA_INCLUDE_DIR})
 
 macro_optional_find_package(OpenGL)
-macro_bool_to_01(OPENGL_FOUND HAVE_OPENGL)
-macro_log_feature(OPENGL_FOUND "OpenGL" "Support for 3D graphs in KAlgebra" "http://mesa3d.sourceforge.net" FALSE "" "")
+option(SHOULD_BUILD_OPENGL "Build OpenGL support" ON)
+if(SHOULD_BUILD_OPENGL)
+  macro_bool_to_01(OPENGL_FOUND HAVE_OPENGL)
+endif(SHOULD_BUILD_OPENGL)
+macro_log_feature(HAVE_OPENGL "OpenGL" "Support for 3D graphs in Kalgebra" "http://opengl.org" FALSE "" "")
 
 macro_log_feature(QTMML_LIBRARY "QtMathML Widget" "Supports displaying some formulas in a mathematic form" "http://doc.qt.nokia.com/solutions/4/qtmmlwidget/qtmmlwidget.html" FALSE "" "")
 
